packages/concerto-tools/lib/codegen/fromcto/golang/golangvisitor.js

```diff
@@ -97,8 +97,8 @@ class GoLangVisitor {
     visitModelFile(modelFile, parameters) {
         // we put all the code into the main package, but we
         // seperate out into multiple files using the namespaces
-        let packageName = this.toGoPackageName(modelFile.getNamespace());
-        parameters.fileWriter.openFile(packageName + '.go');
+        const { mangledNamespace } = ModelUtil.parseNamespace(modelFile.getNamespace());
```

Comment:
I find `mangled` an odd name. What's the purpose of this? To escape `@` for targets that don't allow it?

Reply:
`escapedNamespace`?